packages/components/src/IllustratedMessage/docs/image.tsx

```diff
@@ -0,0 +1,11 @@
+import { Content, Heading, IllustratedMessage, Image } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <IllustratedMessage>
+            <Image src="/frog.jpg" aria-label="No Results" />
```

Comment:
For Images, they need a alt tag, not an aria label. Make sure to correct this everywhere